src/main/java/net/sf/jabref/remote/DBConnector.java

```diff
@@ -0,0 +1,110 @@
+/*  Copyright (C) 2003-2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.remote;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ * Used to establish connections between JabRef and database systems like MySQL, PostgreSQL and Oracle.
+ */
+public class DBConnector {
```

Comment:
It sounds good for few  classes. But `RemoteSynchronizer` would suggest only a remote synchronizing class which isn't one. What about `RemoteDB` or `DBMS`? I've chosen`DBMS` but feel free to suggest some others.